bundles/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/message/LMessage.java

```diff
@@ -55,7 +58,7 @@ public LMessage(String raw) {
     }
 
     public Collection<? extends Device> updateDevices(List<Device> devices, List<DeviceConfiguration> configurations) {
-        byte[] decodedRawMessage = Base64.decodeBase64(getPayload().getBytes(StandardCharsets.UTF_8));
+        byte[] decodedRawMessage = Base64.getDecoder().decode(getPayload().getBytes(StandardCharsets.UTF_8));
```

Comment:
Do we have to trim here too? Similar to line 45?

Reply:
it worked in my testing, but for safety I'll add it anyway.